sfepy/terms/terms_multilinear.py

```diff
@@ -1556,7 +1551,7 @@ class ENonSymElasticTerm(ETermBase):
     def get_function(self, mat, virtual, state, mode=None, term_mode=None,
                      diff_var=None, **kwargs):
         fun = self.make_function(
-            'IK,n(i:j)->I,n(k:l)->K', mat, virtual, state, diff_var=diff_var,
+            'IK,v(i:j)->I,v(k:l)->K', mat, virtual, state, diff_var=diff_var,
```

Comment:
`v(i.j)`, because `:` is reserved for the symmetric gradient. So some tweaks will be needed in the parsing code.

Reply:
> So some tweaks will be needed in the parsing code.

Yes, I already know that :)

Reply:
> `v(i.j)`, because `:` is reserved for the symmetric gradient. So some tweaks will be needed in the parsing code.

Can be `:` used outside `s()`?

Reply:
Can be `s()` used without `:`?